modules/vpc/main.tf

```diff
@@ -0,0 +1,23 @@
+#Create vpc
+resource "aws_vpc" "custom_vpc" {
```

Comment:
There is one argument you need to include, otherwise, when you deploy an RDS instance in this VPC, you won't be able to reach it. `enable_dns_hostname` , see here https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc#enable_dns_hostnames-1

Reply:
I have added this. Added it in the recent pull request